stdlib/string.gr

```diff
@@ -1635,7 +1635,7 @@ let rec decodeRangeHelp =
             // high surrogate. next character is low srurrogate
             let w1 = (w1 & 0x03FFn) << 10n
             let w2 = (WasmI32.load8U(bytesPtr, 2n) << 8n |
-              WasmI32.load8U(bytesPtr, 3n)) &
+                WasmI32.load8U(bytesPtr, 3n)) &
```

Comment:
This seems wrong

Reply:
It does.

Reply:
You might want
(
  WasmI32.load8U(bytesPtr, 2n) << 8n | 
  WasmI32.load8U(bytesPtr, 3n)
 ) &
0x03FFn

to make that clearer.  I can make it do that easily if that's more comprehensible (which I think it is)

Reply:
(imagine the two WasmI32 are indented, the comment isn't showing it)

Reply:
I've made that code change and I think comprehension has jumped up a lot